backend/models/time.py

```diff
@@ -84,7 +84,7 @@ class APITime(Measurer):
 
     def __init__(
             self, access_token='', api_url='',
-            early_exit_outer_radius=120.0 * 10**3,
+            early_exit_outer_radius=200.0 * 10**3,
```

Comment:
what's this?

Reply:
This is another change to get more accurate times in the case of far away points (possibly at the expense of making more API calls). 

Agreed that this should be specified in the class docstring; please add a TODO.

Reply:
Added TODO